src/sidebar/components/menu-keyboard-navigation.js

```diff
@@ -8,27 +8,42 @@ function isElementVisible(element) {
   return element.offsetParent !== null;
 }
 
+/**
+ * @template T
+ * @typedef {import("preact/hooks").Ref<T>} Ref
```

Comment:
This `@typedef` is not needed any more due to the changed way that `useRef` is annotated.